docs/en/latest/concepts/apisix_route.md

```diff
@@ -178,6 +178,12 @@ spec:
 
 APISIX's [80+ Plugins](https://apisix.apache.org/docs/apisix/plugins/batch-requests/) can be used with APISIX Ingress. These Plugins have the same name as in the APISIX documentation.
 
+:::note
+
+If the Plugin is not enabled in APISIX by default, you can enable it by adding it to the `plugins` attribute in your `values.yaml` file while installing APISIX and Ingress controller via Helm.
```

Comment:
I think we can also remind users to directly modify APISIX configuration files, what do you think?

Reply:
Sure. We can add the line "You can also modify your APISIX configuration file directly (`conf/config.yaml`)."

Reply:
great!

Reply:
Updated.

Reply:
Or if it is deployed in Kubernetes, it can be modified through configmap

Reply:
Adding through ConfigMap would mean we are indirectly updating the `config.yaml` file right? Would it be confusing to the user if we mention it in a sentence? There are tutorials that mentions how a user can do that. We can mention these two methods as the easiest and the recommended ways to do that. A user can choose to go the ConfigMap path if they want. What do you think?

Reply:
ok, that make sense